### PR TITLE
Update dataset description and index page for Alma data

### LINKED ISSUE #1704: Update info in GUI about startpage for ALMA
http://lobid.org/resources

### PR DESCRIPTION
**Pull request by fsteeg**
Will resolve #1704.

### CODE REVIEW

**Review by dr0i — APPROVED** [review]

File: web/conf/dataset.jsonld [hunk 1]
```diff
@@ -29,8 +29,8 @@
...
...n-Westfälische Bibliographie und alle Zeitschriften der Zeitschriftendatenbank (ZDB) in den hbz-Verbundkatalog integriert. Die von den Verbundbibliotheken und ihren zugeordneten Einrichtungen aktiv eingebrachten Bestände sind Grundlage für die Online-Fernleihe. Der hbz-Verbundkatalog wird ergänzt um die Bestandsnachweise einer Vielzahl öffentlicher und anderer Bibliotheken, die ihre Bestände maschinell an das hbz liefern.</p><p>Der Verbundkatalog hat einen Umfang von etwa 20 Millionen Titeldatensätzen plus Bestandsinformationen.</p><p>Das hbz, einige Kölner Bibliotheken und das Landesbibliothekszentrum Rheinland-Pfalz starteten 2010 eine Open-Data-Initiative mit der ersten Freigabe von Katalogdaten in Deutschland. Mittlerweile wird der gesamte hbz-Verbundkatalog unter <a href=\"https://creativecommons.org/publicdomain/zero/1.0/\">CC0</a> auf verschiedene Arten bereitgestellt: in dieser Oberfläche, als <a href=\"https://lobid.org/resources/api\">Programmierschnittstelle</a> sowie in Form von Vollabzügen als Linked Open Data und <a href=\"https://lobid.org/download/dumps/DE-605/mabxml/\">MAB/XML</a>. <a href=\"https://lobid.org/resources/dataset\"> Zur Datensetbeschreibung</a>.</p>",
```
Comment (dr0i) [inline web/conf/dataset.jsonld:32]: Better remove the download section. There will be no dump of mab anymore, and maybe even not of MARC at all

File: web/conf/dataset.jsonld [hunk 2]
```diff
@@ -29,8 +29,8 @@
...
+      "en":"<p>The North Rhine-Westphalian Library Service Centre's (hbz) union catalogue records approximately 25 Million bibliographic titles plus holding information. It contains cooperatively created title and holdings information from libraries in North Rhine-Westphalia and Rhineland-Palatinate.</p><p>Besides that, the North Rhine-Westphalian Bibliography (NWBib) and all periodicals from the German Union Catalogue of Serials (ZDB) are included.</p>Cologne-based libraries and the Library Centre of Rhineland-Palatinate (LBZ) in cooperation with the hbz started an open data initiative in 2010 being the first top publish their catalog data under an open license. By now, the whole hbz union catalogue is published under <a href=\"https://creativecommons.org/publicdomain/zero/1.0/\">CC0</a> in different ways and formats: via this user interface, as <a href=\"https://lobid.org/resources/api\">API</a> and as full dumps as <a href=\"https://lobid.org/resources/api#bulk_downloads\">Linked Open Data via the API</a> and as <a href=\"https://lobid.org/download/dumps/DE-605/mabxml/\">downloads of the source data</a>."
```
Comment (dr0i) [inline web/conf/dataset.jsonld:33]: Also it's no more Open Data. This section should be removed.

Comment (acka47) [reply web/conf/dataset.jsonld:33]: Hmm, we will keep publishing lobid data under CC0 but should probably remove the part about the source data for now.

**Comment by fsteeg** [comment]
Updated and redeployed to test:

https://test.lobid.org/resources
https://test.lobid.org/resources/dataset
https://test.lobid.org/resources/dataset.jsonld

**Review by dr0i — APPROVED** [review]
There is still the MAB-xml-download link. But we can leave it as it is. If someone stumbel upon it and complains we will go on.

**Comment by dr0i** [comment]
No, I mean the download URL in `https://github.com/hbz/lobid-resources/blob/master/web/conf/dataset.jsonld#L88`.

**Pull request merged**